### PR TITLE
English version #1

### PR DESCRIPTION
**Pull request by tgmarinho**
If we need to change something in translation just talk. #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,318 +3,308 @@
```
Comment (filipenevola) [inline index.html:3]: Hello, why do we have so many changes? Did you format the code and did the changes? What are the lines with real changes?

Next time do 2 commits, the first with formatting and the second with real changes, ok?

File: index.html [hunk 2]
```diff
@@ -3,318 +3,308 @@
...
+                        <a class="nav-link" href="#projects">Projetos</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#contact">Contato</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://slackincodeftw.herokuapp.com" target="_blank">Chat Online (Slack)</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="index-en.html"> <img src="img/en.png" alt="English Version"></a>
```
Comment (tgmarinho) [inline index.html:70]: new changes here

File: index.html (hunk 1 above)
Comment (tgmarinho) [reply index.html:3]: I think my IDE formatted the code... ok

**Comment by filipenevola** [comment]
@tgmarinho published! Thank you :)

**Pull request merged**